src/main/java/com/shaft/database/DBActions.java

```diff
@@ -0,0 +1,190 @@
+package com.shaft.database;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.testng.Assert;
+
+import com.shaft.io.ReportManager;
+
+/**
+ * 
+ * @author mennamaged
+ *
+ */
+
+public class DBActions {
+
+	private Connection connection;
+	private Statement statement;
+	private ResultSet resultSet;
+	private String dbType;
```

Comment:
recommended to be enum, please open a jira ticket for this as a future enhancement and reply with the link.

Reply:
Done :
https://incorta.atlassian.net/browse/AUTO-32